packages/storybook-vue/src/stories/CustomRadioButton/CustomRadioButton.stories.tsx

```diff
@@ -0,0 +1,27 @@
+import type { Meta, StoryFn } from '@storybook/vue3';
+import { CustomRadioButton } from '@utrecht/component-library-vue';
+import { argTypes } from './argTypes';
+
+export default {
+  component: CustomRadioButton,
+  id: 'vue-custom-radio-button',
+  title: 'Vue.js Component/Custom Radio Button',
+  argTypes: argTypes,
+} as Meta<typeof CustomRadioButton>;
+
+const Template: StoryFn<typeof CustomRadioButton> = (args) => ({
+  components: { CustomRadioButton },
+  setup() {
+    return { args };
+  },
+  template: '<CustomRadioButton v-bind="args" />',
+});
+
+export const Default = Template.bind({});
+Default.parameters = {
+  docs: {
+    source: {
+      code: `<CustomRadioButton />`,
+    },
+  },
+};
```

Comment:
The file name extension for these stories is `.tsx`, but shouldn't it be `.ts`?

Reply:
Honestly, it should be `.vue`, but it's unfortunate I could not find an explanation about Vue.js and Typescript, I don't know if there is a difference between using `.ts` vs `.tsx` with Vuejs,
Should I change them to `.ts` What do you think?